src/libNode/FinalBlockProcessing.cpp

```diff
@@ -691,6 +693,34 @@ void Node::CallActOnFinalblock()
     BroadcastTransactionsToLookup(txns_to_send);
 }
 
+void Node::BroadcastMicroBlockToLookup()
```

Comment:
The suggestion for entire function is: do the check `m_microblock == nullptr` first and have a return if it failed, then proceed with the heavy logic.